<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainPage</class>
 <widget class="QMainWindow" name="MainPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="terminal_reflect">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>351</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser{color:white;background:black}</string>
    </property>
   </widget>
   <widget class="ChoiceWidget" name="choice_widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>371</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>仿真</string>
    </property>
    <addaction name="action_sim"/>
    <addaction name="action_3"/>
    <addaction name="action_temp"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>脚本</string>
    </property>
    <addaction name="action_conf"/>
    <addaction name="action_clean"/>
    <addaction name="action_gen"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>终端</string>
    </property>
    <addaction name="action_terminate"/>
   </widget>
   <addaction name="menu_4"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
   <addaction name="menu_5"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="action_about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_sim">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/img/per_sim_icon.png</normaloff>:/img/per_sim_icon.png</iconset>
   </property>
   <property name="text">
    <string>性能仿真</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>功耗仿真</string>
   </property>
  </action>
  <action name="action_temp">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/img/temp_icon.png</normaloff>:/img/temp_icon.png</iconset>
   </property>
   <property name="text">
    <string>生成温度图</string>
   </property>
   <property name="toolTip">
    <string>生成温度图</string>
   </property>
  </action>
  <action name="action_gen">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/img/gen_icon.png</normaloff>:/img/gen_icon.png</iconset>
   </property>
   <property name="text">
    <string>生成</string>
   </property>
   <property name="toolTip">
    <string>生成脚本</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_conf">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/img/conf_icon.png</normaloff>:/img/conf_icon.png</iconset>
   </property>
   <property name="text">
    <string>配置</string>
   </property>
  </action>
  <action name="action_clean">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/img/clean_icon.png</normaloff>:/img/clean_icon.png</iconset>
   </property>
   <property name="text">
    <string>清理</string>
   </property>
   <property name="toolTip">
    <string>清理脚本</string>
   </property>
  </action>
  <action name="action_terminate">
   <property name="text">
    <string>停止</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChoiceWidget</class>
   <extends>QWidget</extends>
   <header>choicewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
